packages/components/src/utils/testing/snapshot-testing.tsx

```diff
@@ -17,6 +17,9 @@ export const executeSnapshotTests = <Props extends Record<string, unknown>>(Comp
 	describe(ComponentName, () => {
 		const testTable: [string, Props][] = propVariants.map((props) => [propsToLabel(props), props]);
 
+		jest.useFakeTimers();
+		jest.spyOn(global, 'setTimeout');
```

Comment:
fakeTimers kann ich nachvollziehen, scheint in diesem Kontext notwendig zu sein.

Das spyOn hat keinen Effekt, soweit ich es reproduzieren kann. Würde ich gerne entfernen.